src/server/routes/templateRouter.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const authenticate = require('../middlewares/Auth')
+const router=express.Router();
+
+const {fetchTemplates, createTemplate, cloneTemplate, deleteTemplate} = require('./../controllers/templateController');
+
+router.get('/', authenticate, fetchTemplates);
+router.post('/create', authenticate, createTemplate);
+router.post('/clone/:id/', authenticate, cloneTemplate);
```

Comment:
Do we not need an update route for updating an existing template if there be any descrepancy ?

Reply:
On it!